<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>384</width>
    <height>610</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(53, 53, 53, 255), stop:1 rgba(0, 0, 0, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-37</y>
         <width>443</width>
         <height>716</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item alignment="Qt::AlignHCenter|Qt::AlignTop">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>250</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>150</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 40pt &quot;Slime Box&quot;;
color: rgb(0, 130, 195);
</string>
          </property>
          <property name="text">
           <string>Senpai Key</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>239</width>
            <height>106</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>450</width>
            <height>106</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="website_label">
             <property name="minimumSize">
              <size>
               <width>221</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>500</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">#website_label{
font: 36pt &quot;Slime Box&quot;;
color: rgb(227, 227, 227);
}</string>
             </property>
             <property name="text">
              <string>Website:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="website_input">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>221</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">#website_input {
	
	color: rgb(85, 255, 127);
	border: solid;
	border-radius: 15px;
	font: 20pt &quot;Slime Box&quot;;
	border-width: 2px;
	border-color: rgb(227, 227, 227);
	
}

#website_input:focus{
	border-radius: 5px;
	border-color: rgb(0, 170, 255);
	
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="maxLength">
              <number>25</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>239</width>
            <height>106</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>450</width>
            <height>106</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="username_label">
             <property name="minimumSize">
              <size>
               <width>221</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>500</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 36pt &quot;Slime Box&quot;;
color: rgb(227, 227, 227);</string>
             </property>
             <property name="text">
              <string>Username:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="username_input">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>221</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">#username_input {
	color: rgb(85, 255, 127);
	border: solid;
	border-radius: 15px;
	font: 20pt &quot;Slime Box&quot;;
	border-width: 2px;
	border-color: rgb(227, 227, 227);
	
}

#username_input:focus{
	border-radius: 5px;
	border-color: rgb(0, 170, 255);
	
}</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>239</width>
            <height>106</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>450</width>
            <height>106</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="password_label">
             <property name="minimumSize">
              <size>
               <width>221</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>500</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 36pt &quot;Slime Box&quot;;
color: rgb(227, 227, 227);</string>
             </property>
             <property name="text">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="password_input">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>221</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">#password_input {
	color: rgb(85, 255, 127);
	border: solid;
	border-radius: 15px;
	font: 20pt &quot;Slime Box&quot;;
	border-width: 2px;
	border-color: rgb(227, 227, 227);
	
}

#password_input:focus{
	border-radius: 5px;
	border-color: rgb(0, 170, 255);
	
}</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::PasswordEchoOnEdit</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="add_entry_btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>239</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>450</width>
            <height>50</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">#add_entry_btn{
	background-color: rgb(0, 130, 195);
	font: 26pt &quot;Slime Box&quot;;
	color: rgb(227, 227, 227);
	border: solid;
	border-radius: 25px;
	border-width: 2px;
	border-color: rgb(227, 227, 227);
	
}
</string>
          </property>
          <property name="text">
           <string>Add Entry</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="get_entry_btn">
          <property name="minimumSize">
           <size>
            <width>239</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>450</width>
            <height>50</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">#get_entry_btn{	
	background-color: rgb(0, 130, 195);
	font: 26pt &quot;Slime Box&quot;;
	color: rgb(227, 227, 227);
	border: solid;
	border-radius: 25px;
	border-width: 2px;
	border-color: rgb(227, 227, 227);
	
}
</string>
          </property>
          <property name="text">
           <string>Get Entry</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
